### ISSUE TITLE
v2 CHANGELOG.rst fails to render on github.com

### ISSUE THREAD

**Issue by willhughes-au** [aws/aws-cli#7150]
### Describe the bug

When accessing the [AWS CLI v2 CHANGELOG.rst](https://github.com/aws/aws-cli/blob/v2/CHANGELOG.rst) - Github fails to render the page.  
I believe this is because the file has grown over some limits from github. 

### Expected Behavior

https://github.com/aws/aws-cli/blob/v2/CHANGELOG.rst should render correctly.

### Current Behavior

Returns a Github Unicorn error after some period of time. 

```
This page is taking too long to load.

Sorry about that. Please try refreshing and contact us if the problem persists.
```

### Reproduction Steps

Try to navigate to CHANGELOG.rst on Github. Direct link or the file browser (when v2 branch selected). 

### Possible Solution

Trim the file - move older entries to another file. 

### CLI version used

n/a

### Environment details (OS name and version, etc.)

Browser

**Comment by willhughes-au** [comment]
Sorry if this isn't the right way to report a bug with the repo itself.

**Comment by tim-finnigan** [comment]
Thanks @willhughes-au for creating the issue. It's something that occurred recently since the CHANGELOG.rst exceeded a certain size. The team is aware of this and currently looking into solutions for addressing it.

**Comment by tim-finnigan** [comment]
As a temporary workaround, you can refer to this link to the raw file: https://raw.githubusercontent.com/aws/aws-cli/v2/CHANGELOG.rst

**Comment by nateprewitt** [comment]
This should be resolved as of today's release of the CLI v2 ([2.16.6](https://github.com/aws/aws-cli/blob/2.16.6/CHANGELOG.rst)). The CHANGELOG.rst for v2 now only contains entries for CLI v2.0.0 and later, this reduced the size of the file by ~50%. If you still need access to CLI v1 changelog information, it's still available [here](https://github.com/aws/aws-cli/blob/develop/CHANGELOG.rst).
